### ISSUE TITLE
Не работает оплата

### ISSUE THREAD

**Issue by TemKaSD** [yandex-money/yandex-money-ycms-v2-prestashop#35]
Здравствуйте. Настраиваю новый интернет магазин, договор с яндексом заключен, данные шоп ид и секретный ключ верны.

Захожу на страницу выбора метода оплаты
![Screenshot_1](https://user-images.githubusercontent.com/6318282/64303737-cf514a80-cfcc-11e9-95d9-1e090f7a5a94.png)

После отправки формы, просто обновляется страница. Если изменить тип работы модуля, чтобы выбор проходил на яндекс сайте, ошибка: ТЕХНИЧЕСКАЯ error- Ошибка запроса

Версия престы 1.7.6.1 php 7.3.

**Comment by ostulov** [comment]
Добрый день!

В логе модуля нет формирования запроса в Яндекс.Кассу на платеж, только запросы на проверку настроек.

Проверьте оформление заказа без модуля "One Page Supercheckout ", в таком случае переход к оплате происходит?

**Comment by TemKaSD** [comment]
@ostulov Здравствуйте. Да, без модуля работает, спасибо.
@pavel52rus Здравствуйте. Будет добавлена поддержка модуля One Page Supercheckout?

**Comment by ostulov** [comment]
Запросите у разработчика модуля One Page Supercheckout технические подробности, почему не происходит переход к оплате. Если для решения проблемы потребуются изменения на стороне нашего модуля, то мы посмотрим, что можно сделать.

**Comment by TemKaSD** [comment]
@ostulov Не могу, я использую публичную версию без поддержки.

**Comment by ostulov** [comment]
Пришлите на почту cms@yamoney.ru скриншоты настроек модуля One Page Supercheckout. В особенности интересуют вкладки Payment methods, General, Cart, Design.

**Comment by TemKaSD** [comment]
Отправил, но не думаю что это из-за настроек. Могу выслать доступы к фтп и админке.

**Comment by TemKaSD** [comment]
@ostulov В общем, так и не решил проблему самостоятельно (как сказала на почте ваша поддержка) после отправки скринов, проблема актуальна.

**Comment by TemKaSD** [comment]
В последней версии модуля One Page Supercheckout v6.0.1 тоже не работает, просто обновляет страницу.

@ostulov Может вам стоит посмотреть в сторону своего модуля? Ведь с другими модулями оплаты One Page Supercheckout работает, с вашим нет.

Давайте вместе разберемся, я дам доступ к магазину, посмотрите.

**Comment by doom9ra** [comment]
@TemKaSD , приветствую! Не нашли ли решения данной проблемы?

**Comment by TemKaSD** [comment]
> @TemKaSD , приветствую! Не нашли ли решения данной проблемы?

Здравствуйте. Неа. Вы используете One Page Supercheckout?

**Comment by doom9ra** [comment]
@TemKaSD , да, верно Superchekout 6.0.1, страница перезагружается, на страницу оплаты не перенаправляет, товар остаётся в корзине.
Видимо, ошибка под капотом :(

**Comment by TemKaSD** [comment]
> @TemKaSD , да, верно Superchekout 6.0.1, страница перезагружается, на страницу оплаты не перенаправляет, товар остаётся в корзине.
> Видимо, ошибка под капотом :(

Я отошел от этого модуля, он долгий и глючит, были баги с ним.

**Comment by ostulov** [comment]
@doom9ra Добрый день!

Без использования модуля Superchekout 6.0.1 переход к оплате происходит? Запрос в ЮKassa обрабатывается без ошибок?

**Comment by doom9ra** [comment]
@ostulov ,Да,верно, без использования модуля Superchekout 6.0.1 всё хорошо,но к сожалению родной чекаут престы ужасен:)

**Comment by TemKaSD** [comment]
> @ostulov ,Да,верно, без использования модуля Superchekout 6.0.1 всё хорошо,но к сожалению родной чекаут престы ужасен:)

Дайте свой контакт.

**Comment by doom9ra** [comment]
@TemKaSD , doom9ra@mail.ru либо vk.com/doom9ra
